en/peripherals/camera.md

```diff
@@ -48,12 +48,14 @@ On a controller that only has an FMU, the pins map to `MAIN5` and `MAIN6` (e.g.
 At time of writing triggering only works on FMU pins - you can't trigger a camera using pins on the I/O board.
 :::
 
+
 :::warning
```

Comment:
So there is all this old stuff which said you had to use certain outputs for triggering, and that some combinations disabled other pins.
1. Is that all still true when using mixers?>
2. With control allocation, how does this work? I.e. is it still 2 pins you set? 
3. What do we say for camera capture?

Reply:
1. yes for mixers
2. yes for the ones that require 2 pins, you need to assign camera triggering to 2 outputs. Then for the ones that use PWM (e.g. Seagull MAP2), the whole PWM group cannot be used for anything else, you can only have camera triggering in that group.
3. you can also assign any of the outputs

Reply:
> Then for the ones that use PWM (e.g. Seagull MAP2), 

> you can also assign any of the outputs

Don't they all have to use PWM? 

I base this on the yaml which says FMU only https://github.com/PX4/PX4-Autopilot/blob/main/src/lib/mixer_module/output_functions.yaml#L40-L52

![image](https://user-images.githubusercontent.com/5368500/179885097-297301f5-b196-4b3a-8886-5414c8c8a640.png)

Reply:
They use the PWM Pins, but not necessarily PWM signal. For some it's just a GPIO signal that's toggled.

Reply:
@bkueng Thanks. I think I have cleaned this up now. You OK for me to merge?